graylog2-server/src/main/java/org/graylog2/plugin/ServerStatus.java

```diff
@@ -149,6 +149,11 @@ public void overrideLoadBalancerThrottled() {
         publishLifecycle(Lifecycle.OVERRIDE_LB_THROTTLED);
     }
 
+    /**
+     * Blocks until the server enters the RUNNING state and then executes the given Runnable.
+     * <p>
+     * <b>This method is not interruptible while waiting for the server to enter the RUNNING state.</b>
+     */
     public void awaitRunning(final Runnable runnable) {
```

Comment:
We could still mark this one `@Deprecated`
```suggestion
    @Deprecated
    public void awaitRunning(final Runnable runnable) {
```

Reply:
yep, why not